### PR TITLE
Create Collections page

### PR DESCRIPTION
**Pull request by rediris**
Prerequisite: https://github.com/gymnasium/gymcms/pull/574

See https://github.com/gymnasium/tracker/issues/140

### CODE REVIEW

File: lms/templates/static_templates/collections.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%inherit file="../main.html" />
+<%namespace name="gymcms" file="../util/gymcms.mako" />
+
+<%block name="pagetitle">${_("Collections")}</%block>
+
+<%block name="headextra">
+  ${gymcms.render('/meta/collections-meta')}
```
Comment (rediris) [inline lms/templates/static_templates/collections.html]: May change this to `${gymcms.render('/hub-pages/meta/collections-meta')}` or whatever is appropriate, depending on the outcome of the other PR.

**Review by andrewpmiller — APPROVED** [review]
I like it!

**Comment by andrewpmiller** [comment]
collection

File: lms/templates/static_templates/collections.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%inherit file="../main.html" />
+<%namespace name="gymcms" file="../util/gymcms.mako" />
+
+<%block name="pagetitle">${_("Collections")}</%block>
+
+<%block name="headextra">
+  ${gymcms.render('/hub-pages/collections/meta/')}
```
Comment (andrewpmiller) [inline lms/templates/static_templates/collections.html:8]: @rediris this include is generated here: `/static/hub-pages/collections/meta/`

**Pull request merged**